docs/configure-and-manage/manage/qlight-node.md

```diff
@@ -57,11 +58,11 @@ This leverages the security model described under [JSON-RPC security](json-rpc-a
 When using [JSON-RPC security](json-rpc-api-security.md#enterprise-authorization-protocol-integration)
 you must provide an access token to communicate to the qlight server.
 
-Enable auth tokens in the qlight client using `--qlight.client.token.enabled`.
+Enable auth tokens in the qlight client using [`--qlight.client.token.enabled`](../../reference/cli-syntax.md#qlightclienttokenenabled).
 
-Once enabled, specify an initial value using `--qlight.client.token.value <token>`.
+Once enabled, specify an initial value using [`--qlight.client.token.value`](../../reference/cli-syntax.md#qlightclienttokenvalue).
 
-Specify a refresh mechanism for the token using `--qlight.client.token.management`.
+Specify a refresh mechanism for the token using [`--qlight.client.token.management`](../../reference/cli-syntax.md#qlightclienttokenmanagement).
 The valid values are:
 
 - `none` - the token is not refreshed (this mechanism is for development/testing purposes only).
```

Comment:
I would delete this and following two lines since it is described in the linked reference docs (and worded differently there).

Reply:
Updated